assets/app/views/browse/build-config.html

```diff
@@ -22,7 +22,7 @@
                 <button
                     class="btn btn-default hidden-xs"
                     ng-click="startBuild()"
-                    ng-disabled="!canBuild">
+                    ng-disabled="!canBuild()">
```

Comment:
Yeah, I made a similar comment to Jakub that might have led to the bug.

The problem is it's 3 places in each controller you have to update the var, and if you forget one, the button is disabled. I thought it was too easy to get wrong.